flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
why?

Reply:
ARM is not supported by libcpuid: https://github.com/anrieff/libcpuid/issues/96
CPU-X relies mainly on libcpuid, so that is why I prefer to skip ARM.

Reply:
it should be `only-arches` then.

Reply:
Fine, updated in 245ffbcb4d344b13495bd44dd2abb921d38384c8.